lib/prepareNodeESM.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+const FileHound = require('filehound');
+const fs = require('fs');
+
+(async () => {
+  const files = await FileHound.create()
+    .paths(__dirname + '/dist/esm/src')
+    .discard('node_modules')
+    .ext('js')
+    .find((err, files) => {
+      if(err || !files.length){
+        console.error('There is no ESM build folder.');
+      }
+      return files
+    });
+
+  files.forEach((filepath) => {
+    fs.readFile(filepath, 'utf8', (err, data) => {
+      if (!data.match(/(?:import|export) .* from/g)) {
+        return
+      }
+      let newData = data.replace(/((?:import|export) .* from\s+['"])([./].*)(?=['"])/g, '$1$2.js')
```

Comment:
Can you allow multi-line import/export statements by matching on `\n((?:import|export)[\s][^;]*[\s]from\s+['"])([.][/-$\d\w]+)(?=['"])/g, '$1$2.js')`

I think this will work? I haven't tested it, so probably not...

Also, when I did this in the past I did some work to not accidentally double-add `.js` to the end, but I don't remember how to do that...

This will avoid conflicts with prettier and other auto-formatters

Reply:
Because of it runs trough transpiled stuff its ok, tsc removes brakes
![image](https://user-images.githubusercontent.com/41641307/164718612-13d33691-e5e4-4531-aedf-0b7e8e460f81.png)

Reply:
Okay